README.md

```diff
@@ -16,8 +16,21 @@ Bugs should be filed under the Issues section of this repo.
 
 Feature requests can be submitted at https://feedback.mongodb.com/forums/924145-atlas - just select the Terraform plugin as the category or vote for an already suggested feature.
 
-## Requirements
-- [Terraform Version](https://www.terraform.io/downloads.html) While we support minimum of HashiCorp Terraform version 1.0+ (as we have migrated to [Plugin Protocol Version 6](https://developer.hashicorp.com/terraform/plugin/terraform-plugin-protocol#protocol-version-6)), for your safety we strongly recommend only consuming versions of Terraform that are currently receiving Security/Maintenance Updates. For latest Terraform versions see: https://endoflife.date/terraform 
+## Requirements  
+- [HashiCorp Terraform Version](https://www.terraform.io/downloads.html) Compatibility Matrix
+  
+| HashiCorp Terraform Release | HashiCorp Terraform Release Date  | HashiCorp Terraform Full Support End Date  | MongoDB Atlas Support End Date |
```

Comment:
i would probably the table only in one place and make reference to it, e.g. to: https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs

Reply:
makes sense, updated